frontend/src/components/resourceMap/graphViewSlice.tsx

```diff
@@ -59,7 +66,11 @@ export const graphViewSlice = createSlice({
     addKindIcon(state, action: PayloadAction<{ kind: string; definition: IconDefinition }>) {
       state.kindIcons[action.payload.kind] = action.payload.definition;
     },
+    setGlance(state, action: PayloadAction<Glance>) {
+      state.glances[action.payload.id] = action.payload;
```

Comment:
please consider the case when plugin registers multiple glances for the same kind or a node that does not represent a kubeObject at all, maybe we shouldn't have a lookup by kind at all and render custom glances in all nodes, and then based on the node data plugins can decide for themselves if they should render anything or just return null